server/src/main/java/com/cloud/storage/StorageManagerImpl.java

```diff
@@ -1232,6 +1281,30 @@ public void doInTransactionWithoutResult(TransactionStatus status) {
         return deleteDataStoreInternal(sPool, forced);
     }
 
+    @Override
+    public Pair<Map<String, String>, Boolean> getStoragePoolNFSMountOpts(StoragePool pool, Map<String, String> details) {
+        boolean details_added = false;
+        if (details == null) {
```

Comment:
can move this if check, to below if block

Reply:
Done